spec/Section 3 -- Type System.md

```diff
@@ -2167,3 +2167,27 @@ to the relevant IETF specification.
 ```graphql example
 scalar UUID @specifiedBy(url: "https://tools.ietf.org/html/rfc4122")
 ```
+
+### @behavior
+
+```graphql
+directive @behavior(onError: __ErrorBehavior! = PROPAGATE) on SCHEMA
```

Comment:
Thoughts on `directive @onError(behavior: __ErrorBehavior!) on SCHEMA` to mirror the naming of the client `onError` field?

Reply:
Here’s [why I think it should be a request parameter and not a directive](https://github.com/graphql/nullability-wg/discussions/86). (Further, I don’t think we should allow both because that opens up questions around what happens if the client sets one and the user sets the other: who wins?)

Reply:
What about defining the onError behavior only in introspection and not in the SDL? We can then define an abstract "configuration" block that informs execution behavior. Implementations can then choose to make this be defined as a schema directive or as a separate configuration file.